### PR TITLE
docs(container.orchestrator): updated Creating container guide to explain that the name set for PID will be the container name

### PR DESCRIPTION
**Pull request by GregoryIvo**
updated Creating container guide to explain that the name set for PID will be the container name

> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

Brief description of the PR. [e.g. Added `null` check on `object` to avoid `NullPointerException`]

**Related Issue:** This PR fixes/closes {issue number}

**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped.

**Screenshots:** If applicable, add screenshots to help explain your solution

**Any side note on the changes made:** Description of any other change that has been made, which is not directly linked to the issue resolution [e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

**Comment by GregoryIvo** [comment]
Can someone with permission add the labels to back port this to 5.2, 5.1, and 5.0?

File: docs/core-services/container-orchestration-provider-usage.md [hunk 1]
```diff
@@ -21,7 +21,10 @@ To use this service select the **ContainerOrchestrationService** option located
 
 ## Creating your first container.
 
-To create a container, select the + icon (Create a new component) under **services**. A popup dialogue box will appear. In the field **Factory** select **org.eclipse.kura.container.provider.ContainerInstance** from the drop-down. Then, using the **Name** field, enter the name of the container you wish to create and Finally press submit to create the component.
+To create a container, select the + icon (Create a new component) under **services**. A popup dialogue box will appear. In the field **Factory** select **org.eclipse.kura.container.provider.ContainerInstance** from the drop-down. Then, use the **Name** field to specify a name for the container.
```
Comment (mattdibi) [inline docs/core-services/container-orchestration-provider-usage.md]: What about some formatting around the `+` sign?

```suggestion
To create a container, select the `+` icon (Create a new component) under **services**. A popup dialogue box will appear. In the field **Factory** select **org.eclipse.kura.container.provider.ContainerInstance** from the drop-down. Then, use the **Name** field to specify a name for the container.
```

**Review by mattdibi — APPROVED** [review]

**Comment by GregoryIvo** [comment]
Thank you @mattdibi!

**Comment by mattdibi** [comment]
@GregoryIvo whoops, it didn't render correctly. Please fix the rendering issue and then backport manually to the other branches.

image

**Pull request merged**